Player.cs

```diff
@@ -18,7 +18,10 @@ class Player {
 		private int _money;
 		private string _zone;
 		private string _weapon;
-		private int _diedAt;
+		private int _diedAt;
+        private int _vit; //vitality - increase max hp

```

Comment:
Vitality, Strength, and Dexterity aren't ever being set or modified.  They're also not being saved in the SQL table.

NB: Keep in mind, nothing should *only* be kept in RAM.  The game should consider everything as stateless, relying solely upon the SQL data for permanent.  That's why, in the current master branch, every change to Player or Monster or anything is saved, immediately, to SQL.